app/class_room.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :lable
```

Comment:
You seem to have specified attribute accessor but with `label` variable misspelt. Kindly correct it to match the same variable name within the methods to avoid errors.